superset/config.py

```diff
@@ -994,6 +994,8 @@ def CSV_TO_HIVE_UPLOAD_DIRECTORY_FUNC(
 # If set to true no notification is sent, the worker will just log a message.
 # Useful for debugging
 ALERT_REPORTS_NOTIFICATION_DRY_RUN = False
+# Allowed notification methods: Email, Slack
+ALERT_REPORTS_ALERTS_NOTIFICATION_METHODS = ["Email", "Slack"]
```

Comment:
@graceguo-supercat I'm not sure how these types of configuration are handled elsewhere but maybe these should be enums. Also what happens if you make this an empty list?

Reply:
I agree with John, should we disable the feature if the list is empty?

Reply:
Based on the new implementation, we will always has 'Email' option, so i don't handle empty case here.